src/fuzzy_completion.rs

```diff
@@ -0,0 +1,57 @@
+use dialoguer::{
```

Comment:
#766 just removed all this stuff I think. is there a reason we need it?

Reply:
None. I just didn't see the merge. I'll delete it